libfreerdp/core/mcs.c

```diff
@@ -716,6 +722,12 @@ BOOL mcs_send_connect_response(rdpMcs* mcs)
 	Stream_Free(server_data, TRUE);
 
 	return (status < 0) ? FALSE : TRUE;
+
+error_stream_s:
+	Stream_Free(gcc_CCrsp, TRUE);
+error_data_blocks:
+	Stream_Free(s, TRUE);
```

Comment:
That should be Stream_Free(server_data, TRUE); 
s isn't allocated at that time.